### ISSUE TITLE
SyntaxError: Octal escape sequences are not allowed in template strings

### ISSUE THREAD

**Issue by foray1010** [Connormiha/jest-css-modules-transform#34]
Hi, I am using a third-party library with css like below: (I guess for cross browser compatibility)

```css
.main {
  content: '\20';
}
```

when I run test with this css, jest will throw this error:

```
styles.css:3
      content: '\20';
                 ^

    SyntaxError: Octal escape sequences are not allowed in template strings.

      2 | import * as React from 'react'
      3 |
    > 4 | import classes from './styles.css'
```

But it still run in browser, can you advice what can I do to parse this syntax? thank you!

**Comment by Connormiha** [comment]
@foray1010  It is bug. Will be fixed soon at the next release. Thanks.

**Comment by Connormiha** [comment]
@foray1010  Is this code in jest-css-modules-transform? I was unable to reproduce this bug. Can you describe the playback conditions in more detail?

**Comment by foray1010** [comment]
@Connormiha Hi! I have a minimum reproducible example: https://github.com/foray1010/jest-css-modules-transform-bug-example

It only happens when injectIntoDOM: true

**Comment by Connormiha** [comment]
@foray1010 Thanks for your help. It fixed in `v4.2.1`.

PS. Don't forget `jest --clearCache` for first time after update.
